index.html

```diff
@@ -785,7 +785,181 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 		            </div>
 		            
 		        </div>
-		        
+				
+				<div class="module left"></div>
+
+				<section id="contributors" class="contributors-sec">
+					<div id="contributors-list">
+					<div class="container">
+					<div id="contributors-content">
+					
+					<U><center><h2>CONTRIBUTORS</h2></center></U>
+					<a href="https://github.com/fossasia/gci17.fossasia.org/graphs/contributors" target="_blank">
+					<center><h5>Follow them github.com</h5></a></center>
+				 </a>
+					<div class="col-xs-6 col-sm-4 col-md-3 col-lg-2">
+						<div class="card"><div class="avatar">
+							<img src="https://avatars2.githubusercontent.com/u/1583873?s=460&v=4">
+							</div>
+							<a href="https://github.com/mariobehling" target="_blank">
+							<h5><span>Mario Behling</span></div></h5>
+							</a>
+						</div>
+					</div>
+				<div class="col-xs-6 col-sm-4 col-md-3 col-lg-2">
+					<div class="card"><div class="avatar">
+						<img src="https://avatars3.githubusercontent.com/u/14032427?s=460&v=4">
```

Comment:
Don't add contributors statically, use the github api.

Reply:
Closing wrt PR #54.